### DISCUSSION TITLE
No bin file in the installed CAFE5

### DISCUSSION THREAD

**Discussion by Priyanka-uq** [hahnlab/CAFE5#117] [Q&A] [answered]
Hi,
Thanks for designing this great software. 
I have installed CAFE5, but can't see the 'bin' folder inside the CAFE5, below I have pasted the list of the folders. Could you please direct how to give the path in the .bashrc, to run the command?

 4096 Mar  1 15:27 autom4te.cache
-rw-r--r-- 1 uqpshar4 S_QAAFI_CNAFS    380 Mar  1 15:21 CHANGELOG.md
-rw-r--r-- 1 uqpshar4 S_QAAFI_CNAFS   1689 Mar  1 15:21 config.h.in
-rwxr-xr-x 1 uqpshar4 S_QAAFI_CNAFS 114780 Mar  1 15:27 configure
-rw-r--r-- 1 uqpshar4 S_QAAFI_CNAFS   1835 Mar  1 15:21 configure.ac
drwxr-sr-x 4 uqpshar4 S_QAAFI_CNAFS   4096 Mar  1 15:21 docs
drwxr-sr-x 2 uqpshar4 S_QAAFI_CNAFS   4096 Mar  1 15:21 examples
-rw-r--r-- 1 uqpshar4 S_QAAFI_CNAFS    381 Mar  1 15:21 INSTALL
-rw-r--r-- 1 uqpshar4 S_QAAFI_CNAFS   8310 Mar  1 15:21 LICENSE
-rw-r--r-- 1 uqpshar4 S_QAAFI_CNAFS    568 Mar  1 15:21 main.cpp
-rw-r--r-- 1 uqpshar4 S_QAAFI_CNAFS   2383 Mar  1 15:21 Makefile.in
-rw-r--r-- 1 uqpshar4 S_QAAFI_CNAFS  36475 Mar  1 15:21 README.md
drwxr-sr-x 2 uqpshar4 S_QAAFI_CNAFS   4096 Mar  1 15:21 scripts
drwxr-sr-x 2 uqpshar4 S_QAAFI_CNAFS   4096 Mar  1 15:21 src
drwxr-sr-x 2 uqpshar4 S_QAAFI_CNAFS   4096 Mar  1 15:21 src_docs
-rw-r--r-- 1 uqpshar4 S_QAAFI_CNAFS 102251 Mar  1 15:21 test.cpp
drwxr-sr-x 5 uqpshar4 S_QAAFI_CNAFS   4096 Mar  1 15:21 Test_data
uqpshar4@tn311d:.../00004_comparison/011_cafe/CAFE5>

Thanks & Regards,
Priyanka

**Comment by benfulton** [comment, marked as answer]
Hi,

Have you run the following commands in that directory?

```
./configure
make
make install
```

Those are required to install the application.

**Reply by Priyanka-uq** [reply]
Thanks Benfulton, it worked. Thanks!
